src/uwtools/tests/utils/test_file.py

```diff
@@ -37,6 +37,9 @@ def test_StdinProxy():
         assert sp.read() == msg
         # But the stdin proxy can be read multiple times:
         assert sp.read() == msg
+        # And the proxy can be iterated over:
+        for line in sp:
+            assert line == msg
```

Comment:
The trailing-newline fix in `atparse_to_jinja2.py` left a method in `StdinProxy` untested, which this extra assertion now tests.